xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
Yeah, definitely no needed.
Visibly, ffmpeg is also included with double quotes, so I don't think there's an actual rule, here